plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -36,14 +39,15 @@ public static JSONObject generateSchema() {
          * Iterates over the base configurators and adds them to the schema.
          */
         JSONObject schemaConfiguratorObjects = new JSONObject();
-        ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
-        for (Object configuratorObject : configurationAsCode.getConfigurators()) {
+        ConfigurationAsCode configurationAsCodeObject = ConfigurationAsCode.get();
```

Comment:
bad merge I think, revert?

Reply:
Yeah I will just get this to its original state.